src/client-entry.js

```diff
@@ -159,8 +159,11 @@ router.onReady(() => {
 	router.afterEach((to, from) => {
 		// finish loading
 		app.$Progress.finish();
-		// fire pageview
-		app.$fireAsyncPageView(to, from);
+
+		if (!to?.params?.noAnalytics) {
```

Comment:
My suggestion should allow us to not need a new query param. I can't think of an instance wherein we'd want to fire a page view b/c we added a query param to the url...

Reply:
Good point. And as we chatted about how any `#` based nav would also omit pageviews if we do a path comparison. To my knowledge we don't use # based navigation in a way that should fire a pageview event in this app. We use it to move about the page sometimes but that shouldn't generally trigger a new analytics page view.

Reply:
Speaking with Casey, the `noAnalytics` param wouldn't be user facing or shown in the url so we can go forth with this solution and it will potentially be useful elsewhere too.